### PR TITLE
Head

### PR DESCRIPTION
**Pull request by jpagolasf**
Primer parte Proyecto

### CODE REVIEW

**Comment by valtimetrik** [comment]
Pull request deprecated.

File: Onboarding/src/classes/testQuickOrderController.cls [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@isTest
+    private class testQuickOrderController {
+    static testMethod void testQuickOrderController() {
+        Account aTest1 = new Account(Name='test account',Location__Longitude__s=-122.396382, Location__Latitude__s=37.793728);
+        Account aTest2 = new Account(Name='test account');
+        insert aTest1; insert aTest2;
```
Comment (valtimetrik) [inline Onboarding/src/classes/testQuickOrderController.cls]: Reduce the dml operations. Insert a list of Accounts.

File: Onboarding/src/classes/testQuickOrderController.cls [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        
+        Warehouse__c wareTest1 = new Warehouse__c(Name='ware test 1', Street_Address__c='640 Brannan St',City__c='San Francsico', Location__Latitude__s=37.783900,Location__Longitude__s=-122.389332);
+        Warehouse__c wareTest2 = new Warehouse__c(Name='ware test 2', Street_Address__c='160 Brannan St',City__c='San Francsico',Location__Latitude__s=37.775794, Location__Longitude__s=-122.399579);
+        insert wareTest1; insert wareTest2;
```
Comment (valtimetrik) [inline Onboarding/src/classes/testQuickOrderController.cls]: Reduce the dml operations. Insert a list of Warehouse.

File: Onboarding/src/classes/testQuickOrderController.cls [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        
+        Merchandise__c merchTest1 = new Merchandise__c(Name='test iPhone',Price__c=600,Quantity__c=5,Condition__c='New',Warehouse__c=wareTest1.Id);
+        Merchandise__c merchTest2 = new Merchandise__c(Name='test MacBook',Price__c=1000,Quantity__c=10,Condition__c='New',Warehouse__c=wareTest2.Id);
+        insert merchTest1; insert merchTest2;
```
Comment (valtimetrik) [inline Onboarding/src/classes/testQuickOrderController.cls]: Reduce the dml operations.

**Comment by rjulianialtimetrik** [comment]
@valtimetrik is this pull request still valid?

**Comment by valtimetrik** [comment]
Pull request deprecated.

**Pull request closed without merging**